### PR TITLE
[SMALLFIX] Remove invalid Property WORKER_FUSE_ENABLED

### PR DESCRIPTION
**Pull request by YichuanSun**
### What changes are proposed in this pull request?

Now WORKER_FUSE_ENABLED is invalid.

### Why are the changes needed?

improve code quality.

### Does this PR introduce any user facing changes?

no.

### CODE REVIEW

File: dora/core/common/src/main/java/alluxio/conf/PropertyKey.java [hunk 1]
```diff
@@ -6355,10 +6348,11 @@ public String toString() {
       stringBuilder(Name.FUSE_MOUNT_ALLUXIO_PATH)
           .setAlias(Name.WORKER_FUSE_MOUNT_ALLUXIO_PATH)
           .setDefaultValue("/")
+          // TODO(Yichuan): fix the description
           .setDescription(format("The Alluxio path to mount to the given "
-              + "Fuse mount point configured by %s in the worker when %s is enabled "
```
Comment (YichuanSun) [inline dora/core/common/src/main/java/alluxio/conf/PropertyKey.java:6359]: Can I just fix the description like this? @LuQQiu

File: dora/core/common/src/main/java/alluxio/conf/PropertyKey.java [hunk 2]
```diff
@@ -6376,8 +6370,9 @@ public String toString() {
       stringBuilder(Name.FUSE_MOUNT_POINT)
           .setAlias(Name.WORKER_FUSE_MOUNT_POINT)
           .setDefaultValue("/mnt/alluxio-fuse")
-          .setDescription(format("The absolute local filesystem path that worker (if %s is enabled)"
-              + "or standalone Fuse will mount Alluxio path to.", Name.WORKER_FUSE_ENABLED))
+          // TODO(Yichuan): Please fix the description
+          .setDescription("The absolute local filesystem path that worker "
+              + "or standalone Fuse will mount Alluxio path to.")
```
Comment (YichuanSun) [inline dora/core/common/src/main/java/alluxio/conf/PropertyKey.java:6375]: ditto

**Review by YichuanSun — COMMENTED** [review]
Remain two questions.

File: dora/core/common/src/main/java/alluxio/conf/PropertyKey.java (hunk 1 above)
Comment (LuQQiu) [reply dora/core/common/src/main/java/alluxio/conf/PropertyKey.java:6359]: please remove the "in the worker"
now it's not related to worker anymore

**Review by LuQQiu — APPROVED** [review]

**Comment by YichuanSun** [comment]
alluxio-bot, merge this please.

**Pull request merged**